src/Installers/Windows/AspNetCoreModule-Setup/.gitignore

```diff
@@ -0,0 +1,3 @@
+Debug/
```

Comment:
@jkotalik your changes seemed to merge cleanly. But, please confirm the updates in 'release/2.2' are _correct_.

Reply:
Wait wait wait... This PR looks wrong. I never committed to 2.1 (only 2.2). This PR seems to have picked up a few commits from 2.2.

Btw I think the instructions for updating this pull request need to be fixed. I think the last line should be a force push.